content/300-accelerate/900-compare.mdx

```diff
@@ -0,0 +1,97 @@
+---
+title: 'Compare Accelerate'
+metaTitle: 'Comparing Prisma Accelerate to other connection pooling options'
+metaDescription: 'Learn how Prisma Accelerate compares to other connection poolers like pgbouncer.'
+hide_table_of_contents: true
+---
+
+<TopBlock>
+
+Prisma Accelerate supports products that serve a global audience, with a global caching system and connection pool that spans multiple regions, providing consistent access to data with low latency no matter where your user (or your database) is located in the world.
+
+The managed connection pool is designed to support serverless infrastructure, capable of handling high volumes of connections and adapting to traffic spikes with ease.
+
+Explore how Prisma Accelerate compares to other global cache and connection pool solutions on the market, and discover what sets it apart.
+
+</TopBlock>
+
+## Accelerate global cache
+
+Prisma Accelerate offers a powerful global cache, so you can serve data to your users at the edge — the closest point to where the users are located — no matter where your database is hosted. This not only speeds up the experience for users, but also reduces read load on your database as well by avoiding roundtrips.
+
+|  | Accelerate | Hyperdrive | Planetscale Boost |
+| --- | --- | --- | --- |
+| **Fully Managed** | ✅ | ✅ | ✅ |
+| **Globally distributed edge infra** | ✅ | ✅ | ✅ |
+| **Control cache policy from code** | ✅ | ❌ | ❌ |
+| **Query-level cache policies** | ✅ | ❌ | ❌ |
+| **Authenticate with API key** | ✅ | ❌ | ❌ |
+| **Postgres compatible** | ✅ | ✅ | ❌ |
+| **MySQL compatible** | ✅ | ❌ | ✅ |
+| **MongoDB compatible** | ✅ | ❌ | ❌ |
+| **Automatic cache updates** | ❌ | ❌ | ✅ |
+
+**Why are these important?**
+
+- Integration with the Prisma client is important for existing Prisma ORM users — it means you can control caching policies directly from your existing codebase with just an extra line of code. Integration is seamless. Here is an example using [the stale-while-revalidating caching strategy](/accelerate/caching#stale-while-revalidate-swr):
+    
+    ```jsx
+    await prisma.user.findMany({
+      cacheStrategy: {
+        swr: 60,
+      },
+    });
+    ```
+    
+- Query level cache policies are critical for serious applications, so that you can control which queries are cached, and the characteristics of the policy. You may want certain data in your app to be cached for several days, other data to be cached for a just a few minutes, and other data to be not cached at all. This is only possible with Prisma Accelerate.
```

Comment:
I honestly think this is such a huge point that people over look. I am glad you are highlighting this. I almost want it sooner in the page but I'm not sure where I would put it.

Reply:
I've added a mention to it in the introductory section now -- I agree it's a biggy.